microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/MicroProfileWorkspaceService.java

```diff
@@ -39,4 +55,36 @@ public void didChangeWatchedFiles(DidChangeWatchedFilesParams params) {
 
 	}
 
+	@Override
+	public CompletableFuture<Either<List<? extends SymbolInformation>, List<? extends WorkspaceSymbol>>> symbol(
+			WorkspaceSymbolParams params) {
+		return javaTextDocuments.getWorkspaceProjects() //
```

Comment:
We need really having a cancelChecker here since search can be long to do and if user quit the search.

Reply:
https://github.com/eclipse/eclipse.jdt.ls/issues/2415